internal/saucecloud/cloud.go

```diff
@@ -1000,8 +997,16 @@ func (r *CloudRunner) reportSuiteToInsights(res result) {
 		return
 	}
 
-	var testRuns []insights.TestRun
+	// read job from insights to get accurate platform and device name
+	j, err := r.InsightsService.ReadJob(context.Background(), res.job.ID)
+	if err != nil {
+		log.Err(err).Msgf(msg.FailedToReadJob, res.job.ID)
```

Comment:
Telling the user "unable to read the job" here is not that meaningful. In what context? What's the consequence?
If you are indeed intending to exit right here (which you do in the next line), then you might as well print `msg.InsightsReportError`. Because that's the consequence. The reason is in `log.Err(err)`.